app/models/entry.rb

```diff
@@ -15,4 +15,6 @@ class Entry < ApplicationRecord
   after_initialize do
     self.github_url ||= "https://github.com/rails-hackathon/#{team.repo_name}"
   end
+  
+  attribute :year, default: Time.current.year
```

Comment:
We should hardcode this to the current event year so it doesn't accidentally change.

Now that I'm thinking about this, we should maybe not use `year` and should instead use a date in case we have multiple events per year. @cjilbert504

Reply:
Perfect. lol